main/ControllersPresenters/InterviewLevel.java

```diff
@@ -34,6 +34,20 @@ public String getInterviewOutput(String input) throws Exception{
             return getEndOfInterviewPrompt();
```

Comment:
This levelEnded() will never evaluate to true if you never use the method endLevel() somewhere.